cmd/podman/root.go

```diff
@@ -128,6 +128,18 @@ func persistentPreRunE(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	for _, env := range cfg.Engine.Env {
+		splitEnv := strings.SplitN(env, "=", 2)
+		key := splitEnv[0]
+		value := ""
+		if len(splitEnv) == 2 {
+			value = splitEnv[1]
+		}
```

Comment:
If it is not 2, this should also be an error.

Reply:
I see what you did, but users might accidentally override an existing environement variable with nothing.
If they want nothing then they should set it 
foo=""